src/util/regexp/dash-g.js

```diff
@@ -4,14 +4,14 @@ define(function() {
  * Generated by:
  *
  * var regenerate = require( "regenerate" );
- * var dashSymbols = require( * "unicode-8.0.0/General_Category/Dash_Punctuation/symbols" );
+ * var dashSymbols = require( "https://github.com/node-unicode/unicode-13.0.0/General_Category/Dash_Punctuation/symbols" );
  * regenerate().add( dashSymbols ).toString();
  *
  * https://github.com/mathiasbynens/regenerate
- * https://github.com/mathiasbynens/unicode-8.0.0
+ * https://github.com/node-unicode/unicode-13.0.0
  *
  * NOTE: In addition to [:dash:],  the below includes MINUS SIGN U+2212.
  */
-return /[\-\u058A\u05BE\u1400\u1806\u2010-\u2015\u2E17\u2E1A\u2E3A\u2E3B\u2E40\u301C\u3030\u30A0\uFE31\uFE32\uFE58\uFE63\uFF0D\u2212]/g;
+return /[\x2D\u058A\u05BE\u1400\u1806\u2010-\u2015\u2E17\u2E1A\u2E3A\u2E3B\u2E40\u301C\u3030\u30A0\uFE31\uFE32\uFE58\uFE63\uFF0D\u2212]|\uD803\uDEAD/g;
```

Comment:
I've update the rest of regexes using the latest `unicode` package.
Note that `-` == `\x2D` here.